src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -421,7 +421,11 @@ private static int GetFirstStatementAffectedIndex(SyntaxNode innermostCommonBloc
             if (precedingEndOfLine == default)
             {
                 return oldStatements.ReplaceRange(
-                    nextStatement, new[] { newStatement, nextStatement });
+                    nextStatement, new[]
+                    {
+                        newStatement.WithLeadingTrivia(nextStatementLeading),
+                        nextStatement
```

Comment:
💭 I'm concerned about what's going to happen to the leading trivia of `nextStatement`. Is it going to show up twice? I'd like to see some tests added that include comment trivia.